### ISSUE TITLE
Output Node Names for the pre-trained data ?

### ISSUE THREAD

**Issue by dangithubs** [ry/tensorflow-resnet#29]
I am trying to use Freeze_Graph.py in Tensorflow_serving on the pre-trained data listed on this page 

I have tried the output node names as  "Accuracy/predictions"

However this is not correct, I expect the output node names were passed into whatever trained the model. However they this isn't listed in the Readme.md.

**Comment by amejs** [comment]
Hello @dangithubs 

How are you loading the pre-trained weights into the model? I would like to finetune it for a different task. Thanks in advance
